src/cargo/ops/cargo_new.rs

```diff
@@ -17,11 +17,13 @@ use toml;
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub enum VersionControl { Git, Hg, NoVcs }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Yeah I suspect the changes below would remove the need for `Clone`, and otherwise I personally prefer to minimize the number of derivations needed to reduce codegen/compile time.